main/rpc/index.js

```diff
@@ -227,6 +229,20 @@ const rpc = {
   resolveAragonName (name, chainId, cb) {
     resolveName(name, chainId).then(result => cb(null, result)).catch(cb)
   },
+  resolveEnsName (name, cb) {
+    log.info('resolving name ', name)
+    const nebula = nebulaApi()
+    nebula.ens.resolve(name).then(result => {
+      const {addresses: {eth: ethAddress}} = result
+      if(!ethAddress){
+        throw new Error(`No ETH address found for ENS domain ${name}`)
+      }
+      return cb(null, ethAddress)
+    }).catch(err => {
+      log.debug('Unable to complete ENS resolution', {name, err})
```

Comment:
this logging is a little bit cryptic and might be confusing when casually scrolling through the logs, maybe something like
```
log.debug(`Could not resolve ENS name ${name}`, err)
```

would be a little more informative